app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= image_tag attachments['pieFullTanLogo.svg'].url, width: '150', align: "middle" %>
+
+<p>Hi<%= @greeting_name ? " #{@greeting_name}," : ","%></p>
+
+<p>Welcome to Pie for Providers! Get ready to increase your slice of the pie with simple child care subsidy management tools. To get started, please verify your email address by clicking the link below (you can also copy and paste it into your browser).</p>
+
+<p><%= link_to 'Confirm Account', "#{@front_end_path}/confirm?confirmation_token=#{@resource.confirmation_token}", target: "_blank" %></p>
```

Comment:
Should use a helper to construct the url or have it constructed by the mailer and saved as an `@` variable to use in the view